### PR TITLE
feat: move full eval to floating toolbar

### PR DESCRIPTION
**Pull request by dhess**
This needs a lot of styling work, and the floating toolbar window needs to be larger, but it's probably better than the status quo, overall. I think it'll look better once we eliminate the sidebar, and also if we break the buttons in the toolbar out into a separate component. See https://github.com/hackworthltd/primer-app/issues/950 for my thoughts on that, and more.

### CODE REVIEW

File: src/Edit.tsx [hunk 1]
```diff
@@ -218,16 +218,16 @@ const AppNoError = ({
...
   return (
     <div className="grid h-screen grid-cols-[18rem_auto_20rem]">
       <div className="h-full overflow-hidden">
         <Sidebar
           initialMode="T&D"
           prog={{
-            defs: p.module.defs
-              .sort((a, b) => cmpName(a.name, b.name))
-              .map((d) => d.name.baseName),
+            defs: defs,
```
Comment (georgefst) [inline src/Edit.tsx:230]: Not important, but I'm a bit surprised that ESLint doesn't recommend simplifying this to `defs,`.

File: src/primer-api/index.ts [hunk 2]
```diff
@@ -1,3 +1,2 @@
 export * from './primer-api';
 export * from './model';
-
```
Comment (georgefst) [inline src/primer-api/index.ts]: Any idea why this churn keeps happening?

File: src/Edit.tsx [hunk 3]
```diff
@@ -313,7 +306,13 @@ const AppNoError = ({
               }}
               initialMode="tree"
               // Note: these offsets are rather arbitrary.
-              initialPosition={{ x: canvasDimensions.width - 100, y: 15 }}
+              initialPosition={{ x: canvasDimensions.width - 410, y: 10 }}
```
Comment (georgefst) [inline src/Edit.tsx:309]: This concerns me a little. I think we do have an issue somewhere about removing this. But until then, perhaps a comment about what these arbitrary numbers are intended to achieve?

**Review by georgefst — APPROVED** [review]

File: src/Edit.tsx (hunk 3 above)
Comment (dhess) [reply src/Edit.tsx:309]: They've been there all along, of course, and I'm just changing the offset here. Anyway, this is just a hack to get the initial position of the window in the bounds of the canvas.

This is fixed slightly more robustly in an upcoming PR.

File: src/primer-api/index.ts (hunk 2 above)
Comment (dhess) [reply src/primer-api/index.ts]: No.

File: src/Edit.tsx (hunk 1 above)
Comment (dhess) [reply src/Edit.tsx:230]: Right, I will fix that.

Comment (dhess) [reply src/Edit.tsx]: Fixed.

**Pull request merged**